README.md

```diff
@@ -16,7 +16,7 @@ This REDCap module generates a PDF of a survey upon completion and saves it to a
 
 Before you can configure this module, you will need to create the surveys and fields needed for configuration. This module can save PDFs for as many surveys as desired, but each survey is saved as its own PDF. The PDFs must be saved to file upload fields in a non-survey instrument. On that instrument create a file upload field for each survey that needs to be saved.
 
-Should you need to save multiple revisions of a survey PDF for a single record\_id, use the Online Designer to copy the file upload file as many times as you need. This module will use the original field name as a basename and append '\_1', '\_2', '\_3', etc. looking for additional fields into which it can save the PDF. Also, set the source instrument as a repeating instrument so that the survey can be filled out multiple times.
+Should you need to save multiple revisions of a survey PDF for a single record\_id, use the Online Designer to copy the file upload file as many times as you need. This module will use the original field name as a basename and append '\_1', '\_2', '\_3', etc. looking for additional fields into which it can save the PDF. If your original field name has the following format `field_name_1`, the module will increment the number at the end of field name by `1` instead of appending '\_1', e.g. `target_field_3` will become `target_field_4` if `target_field_3` is unavailable.
```

Comment:
Line 82 throws an error when the script runs out of writable file upload fields to write to:

> ERROR: There is an error in the REDCap plugin that you are currently using. Please contact this plugin's developer and send them the error info listed below so that they may fix this issue. If you do not know who the developer is for this plugin, then notify your REDCap administrator about this issue.

Error message: Uncaught Error: Call to undefined function Html2Text\mb_detect_encoding() in /var/www.backup/redcap-20171208T1353/redcap_v8.0.3/Libraries/Html2Text.php:47 Stack trace: #0 /var/www.backup/redcap-20171208T1353/redcap_v8.0.3/Classes/Message.php(159): Html2Text\Html2Text::convert('ERROR: REDCap f...') #1 /var/www.backup/redcap-20171208T1353/modules/save_survey_pdf_to_field_v1.0/utility.php(181): Message->send() #2 /var/www.backup/redcap-20171208T1353/modules/save_survey_pdf_to_field_v1.0/ExternalModule.php(82): sendEmail('pbc@ufl.edu', 'please-do-not-r...', NULL, 'ERROR: PDF of R...', 'ERROR: REDCap f...', '/var/www.backup...') #3 [internal function]: SaveSurveyPdfToFieldModule\ExternalModule\ExternalModule->hook_survey_complete('16', '9', 'prescreening_su...', '53', NULL, '8pr8VQaunG', '14', '4') #4 /var/www.backup/redcap-20171208T1353/external_modules/classes/ExternalModules.php(1160): call_user_func_array(Array, Array) #5 /var/www.backup/redcap-20171208T1353/external_modules/classes/ExternalModules.php(1220): 
File: /var/www.backup/redcap-20171208T1353/redcap_v8.0.3/Libraries/Html2Text.php 
Line: 47